packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -123,6 +124,15 @@ export enum PLATFORM {
   Server,
 }
 
+// APFv6 does not have single FESM anymore. Instead of verifying if we're pointing to
+// FESMs, we resolve the `@angular/core` path and verify that the path for the
+// module starts with it.
+// This may be slower but it will be compatible with both APF5, 6 and potential future
+// versions (until the dynamic import appears outside of core I suppose).
+// We resolve any symbolic links in order to get the real path that would be used in webpack.
+const angularCorePackagePath = require.resolve('@angular/core/package.json');
+const angularCoreResourceRoot = fs.realpathSync(path.dirname(angularCorePackagePath));
```

Comment:
`realpath` might or might not play well with bazel and `preserve-symlink`. Just want to make sure it's taken into account.

Reply:
I think it should be ok since it's the realpath to a file that was already resolved via require.resolve (thus using Bazel resolution). It's also the same logic as before though, so this change isn't breaking that.